src/main/java/com/pikachu/purple/domain/rating/Rating.java

```diff
@@ -11,19 +12,25 @@ public class Rating {
 
     private Long ratingId;
     private Long userId;
-    private Long perfumeId;
+    private Long reviewId;
     private Double score;
+    private EntityStatus entityStatus;
 
     @Builder
     public Rating(
         Long ratingId,
         Long userId,
-        Long perfumeId,
+        Long reviewId,
         Double score
     ){
         this.ratingId = ratingId;
         this.userId = userId;
-        this.perfumeId = perfumeId;
+        this.reviewId = reviewId;
+        this.score = score;
+        this.entityStatus = EntityStatus.ACTIVE;
+    }
+
+    public void updateScore(Double score) {
         this.score = score;
```

Comment:
p2) null 이 들어왔을 때 조치가 필요할 것 같아요
```suggestion
        this.score = (score != null) ? score : this.score
```

Reply:
좋습니다!